### PR TITLE
Added executors - part 1

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added executors - part 1

### CODE REVIEW

**Review by andreibancioiu — DISMISSED** [review]
:+1:

File: executors/errors.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package executors
+
+import "errors"
+
+var (
+	errNilRatingsChecker             = errors.New("nil ratings checker instance")
+	errNilOutputNotifier             = errors.New("nil output notifier")
+	errNilValidatorStatisticsQuerier = errors.New("nil output notifier")
```
Comment (miiu96) [inline executors/errors.go:8]: change error message

File: executors/blsKeysExecutor.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	log.Debug("blsKeysExecutor.notify", "executor", executor.name, "num messages", len(messages))
+
+	executor.statusHandler.ProblematicBLSKeysFound(messages)
+	for _, notifier := range executor.notifiers {
+		notifier.OutputMessages(messages...)
+	}
+}
+
+func shortIdentifier(identifier string) string {
+	if len(identifier) <= 15 {
```
Comment (miiu96) [inline executors/blsKeysExecutor.go:138]: maybe you can add a const for this magic number

File: executors/blsKeysExecutor.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+		return identifier
+	}
+
+	return identifier[:6] + "..." + identifier[len(identifier)-6:]
```
Comment (miiu96) [inline executors/blsKeysExecutor.go:142]: also here

File: executors/errors.go (hunk 1 above)
Comment (iulianpascalau) [reply executors/errors.go:8]: done

File: executors/blsKeysExecutor.go (hunk 2 above)
Comment (iulianpascalau) [reply executors/blsKeysExecutor.go:138]: refactored

File: executors/blsKeysExecutor.go (hunk 3 above)
Comment (iulianpascalau) [reply executors/blsKeysExecutor.go:142]: refactored

**Review by miiu96 — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**